aion/web/models/question.ex

```diff
@@ -37,10 +37,26 @@ defmodule Aion.Question do
   end
 
   @spec get_random_question(integer) :: Question.t
-  def get_random_question(category_id) do
-    query = from q in Question, where: q.subject_id == ^category_id
-    query
-    |> Repo.all()
-    |> Enum.random()
+  def get_random_question(room_id) do
+    query = Repo.query("
+      SELECT content, image_name, q.id FROM questions AS q
+      JOIN room_categories AS rc ON rc.category_id = q.category_id
```

Comment:
Be explicit about the join type. You want it to be `INNER JOIN` here. Also keep to one statement per line, so you can move the `ON` to a separate line.

Reply:
I feel like `JOIN` and `ON` should be in one line.

Reply:
Then use one of [these](https://solutioncenter.apexsql.com/sql-formatting-standards-joins-lists-structure-operations/) approaches maybe